gister.sh

```diff
@@ -312,8 +312,8 @@ sync_gist() {
               echo "DIRTY $gist_id"
             else
               echo "DIRTY $gist_id, auto committing"
-              git add . && git commit -m "automated update" && \
-                git pull > /dev/null && git push > /dev/null
+              git add . && git commit -m "automated update"
```

Comment:
I think here `&&` is necessary. If for some reason `git commit` fails, we should not pull/push.